pkg/chart/ChartService.go

```diff
@@ -628,16 +628,16 @@ func (impl ChartServiceImpl) getChartMetaData(templateRequest TemplateRequest) (
 	}
 	return metadata, err
 }
-func (impl ChartServiceImpl) getRefChart(templateRequest TemplateRequest) (string, string, error, string) {
+func (impl ChartServiceImpl) getRefChart(templateRequest TemplateRequest) (string, string, error, string, string) {
 	var template string
 	var version string
-
+	var pipelineStrategyPath string
```

Comment:
write down little desc in comment what exactly it contains.

Reply:
done